examples/hello-world/hello-world/schema/hello_world.schema.graphql

```diff
@@ -1,12 +1,36 @@
-type Block @entity {
+type SubExampleEntity @entity {
```

Comment:
Can we leave the `hello-world` example as is? Ideally the only indexer that gets update from this is just our test indexer